spec/serializers/api/v2/subheadings/subheading_serializer_spec.rb

```diff
@@ -0,0 +1,40 @@
+RSpec.describe Api::V2::Subheadings::SubheadingSerializer do
+  subject(:serializer) { described_class.new(serializable).serializable_hash.as_json }
+
+  let(:serializable) do
+    path = Rails.root.join(file_fixture_path, 'cached_subheading_exhaustive_annotated.json')
+    file = File.read(path)
+    file = JSON.parse(file)
```

Comment:
a small thing, better not use the same name for different things,
since `JSON.parse(file)` does not return a file:
```
data_hash = JSON.parse(file)
Hashie::Mash.new(data_hash)
```
or any other name ;) 



Hashie::Mash.new(JSON.parse(file))